src/selection_stream/selection_stream_processor.ts

```diff
@@ -128,15 +128,16 @@ export class SelectionStreamProcessor
    */
   selectZone(
     anchor: AnchorZone,
-    mode: SelectionEvent["mode"] = "overrideSelection"
+    mode: SelectionEvent["mode"] = "overrideSelection",
+    eventType: SelectionEvent["type"] = "ZonesSelected"
```

Comment:
I don't think it's a good idea to add `eventType` to this method's API.
Especially given it's a public method but the extra argument is only used internally (from an external POV, there's no such thing as an event)

Reply:
Didn't realize the method was public, you're right it doesn't make sense to expose that